src/en_us/index.js

```diff
@@ -128,9 +151,17 @@ export const i19creditorFeesOperational = 'Operational fee'
 export const i19currencyId = 'Currency Identifier'
 export const i19currencySymbol = 'Currency symbol'
 export const i19custom = 'Customize'
+export const i19customizations = 'Customizations'
+export const i19customizationsAddToPrice = 'Price alteration due to this customization'
+export const i19customizationsAttachment = 'URL of file attached by customer to this field'
+export const i19customizationsLabel = 'Label of customization'
```

Comment:
Acho que deveria ser um dos dois aí:

```suggestion
export const i19labelOfCustomization = 'Label of customization'
```

```suggestion
export const i19customizationSLabel = 'Customization\'s label'
```